installer/install-graphql-language-server.cmd

```diff
@@ -0,0 +1,14 @@
+@echo off
+
+if not exist package.json (
+	call npm init -y
+
+	echo {"name":""}>package.json
+)
+
+call npm install "graphql"
+call npm install "graphql-language-service-cli"
+
+echo @echo off ^
+
+call %%~dp0\node_modules\.bin\graphql-lsp %%* graphql-language-server

```

Comment:
I want to do `ln -s "./node_modules/.bin/graphql-lsp" graphql-language-server`.
But I'm not shure this is correct...